src/commcare_cloud/ansible/roles/pgbouncer/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
+postgresql_port: 5432
```

Comment:
sucks that this is needed here. What do you think about removing it from the pgbouncer.ini since pgbouncer will use 5432 if it's not provided (and we always use the standard port)?

Reply:
will test and implement.